how-to-retrieve-cluster-yaml-from-custom-cluster/cluster-yaml-recovery.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+if [[ ! -f kube_config_cluster.yml ]]
+then
+  echo "The kube_config_cluster.yml is missing"
+  exit 1
+fi
+
+if [[ -f cluster.yml ]]
+then
+  echo "cluster.yml exists, please move or rename this file."
+  exit 1
+fi
+
+echo "Checking that jq is installed"
+if [ -x jq ]
+then
+  echo "Please download jq from https://github.com/stedolan/jq/releases/tag/jq-1.6 and install it"
+  exti 1
+fi
+
+echo "Checking that yq v3.x is installed"
+if ! `yq -V | grep 'yq version 3'`
```

Comment:
The output of the command will be considered as another command and the script tries to execute it and fails.
Need to rewrite like below

```
if [  -z  "`yq -V |grep "yq version 3"`" ]
```

```
+ [[ ! -f kube_config_cluster.yml ]]
+ [[ -f cluster.yml ]]
+ echo 'Checking that jq is installed'
Checking that jq is installed
+ '[' -x jq ']'
+ echo 'Checking that yq v3.x is installed'
Checking that yq v3.x is installed
++ grep 'yq version 3'
++ yq -V
+ yq version 3.4.1
Error: unknown command "version" for "yq"
Run 'yq --help' for usage.
+ echo 'Please download yq v3.x from https://github.com/mikefarah/yq/releases/tag/3.4.1 and install it'
Please download yq v3.x from https://github.com/mikefarah/yq/releases/tag/3.4.1 and install it
+ exit 1
```

Reply:
Fixed